### ISSUE TITLE
请问在算g (gradient)这个tensor的时候需不需要tf.stop_gradient？

### ISSUE THREAD

**Issue by zteddy** [GXYM/GHM_Loss#5]
看到原paper的pytorch的实现里算g的时候用了detach。
https://github.com/libuyu/mmdetection/blob/23f86276b6e135415a411aa229e16a9981fac907/mmdet/models/losses/ghm_loss.py#L73
`g = torch.abs(pred.sigmoid().detach() - target)`
想问一下在tensorflow里是不是要用对应的tf.stop_gradient？
我有一点理不清在算GHM_loss weight的这个graph branch会不会有gradient。
